package.json

```diff
@@ -158,6 +156,8 @@
     "jest": "^29.5.0",
     "jest-junit": "^15.0.0",
     "knex": "^2.4.2",
+    "lodash.difference": "^4.5.0",
```

Comment:
I tihnk this should trigger a yarn lock update.. did you run `yarn install` again

Reply:
```
% yarn install
yarn install v1.22.19
[1/5] 🔍  Validating package.json...
[2/5] 🔍  Resolving packages...
success Already up-to-date.
```

Reply:
it does not. yarn.lock doesn't manage `dependancies` versus `devDependancies`, just the version number of any dependancies